src/univariates.jl

```diff
@@ -551,8 +522,8 @@ function _pdf!(r::AbstractArray, d::DiscreteUnivariateDistribution, X::UnitRange
     return r
 end
 
-
-pdf(d::DiscreteUnivariateDistribution) = isbounded(d) ? pdf(d, minimum(d):maximum(d)) :
+# TODO: should we get rid of this?
```

Comment:
If the user calls it on a distribution that doesn't have finite support, that's on them. 😛